packages/fluentui/react-northstar/src/components/Avatar/Avatar.tsx

```diff
@@ -98,6 +103,15 @@ const Avatar: React.FC<WithAsProp<AvatarProps>> & FluentComponentStaticProps<Ava
           }),
       })}
       {!image &&
```

Comment:
In this case there should be a same condition for `image`, i.e. `!icon && Image.create`.

I suggest: 

```
const imageElement = Image.create()
const iconElement = Icon.create()

const hasGlyph = !!image || !!icon

return (
  <JSX>
    {hasGlyph && (imageElement || iconElement)} 
    {!hasGlyph && Label.create()}
  <JSX>
)
```